R/CDPRemote.R

```diff
@@ -35,19 +36,21 @@ NULL
 #'     connect to the remote application.
 #' * `max_attempts`: Logical scalar, number of tries to connect to headless
 #'     Chromium/Chrome.
+#' * `target_id`: A character scalar, identifier of the target. For advanced
+#'     use only.
 #' * `callback`: Function with one argument, executed when the R session is
 #'     connected to the remote application. The connection object is passed
 #'     to this function.
 #'
 #' @section Details:
 #' `$new()` declares a new remote application.
 #'
-#' `$connect(callback = NULL)` connects the R session to the remote application.
-#' The returned value depends on the value of the `callback` argument. When
-#' `callback` is a function, the returned value is a connection object. When
-#' `callback` is `NULL` the returned value is a promise which becomes fulfilled
-#' once R is connected to the remote application. Once fulfilled, the value of
-#' this promise is the connection object.
+#' `$connect(target_id = "default", callback = NULL)` connects the R session to
```

Comment:
As `target_id` is for advanced use only, do you think it will be more use thant `callback` ? 
I am thinking about changing to `.target_id` and to last argument... 
Just a thought...

Reply:
Even if `callback` is usually the last argument, your proposal makes sense.

Reply:
done in https://github.com/RLesur/crrri/pull/56/commits/1ef67194d4d81c57d38b985d4fa7fd4ef2052a30